notebooks/COS/CalCOS/CalCOS.ipynb

```diff
@@ -8,39 +8,39 @@
     "\n",
```

Comment:
<p>Hi Sierra,</p><p>including an enumeration for the loop in the code is unnecessary. Instead, we could achieve the same result with the following:</p><p> </p><pre class="ql-syntax" spellcheck="false">for rawfile in rawfiles:
&nbsp; &nbsp; if verbose:
&nbsp; &nbsp; &nbsp; &nbsp; print(rawfile)

&nbsp; &nbsp; # Read that rawfiles header
&nbsp; &nbsp; header = fits.getheader(rawfile)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>